build.gradle

```diff
@@ -513,6 +520,17 @@ subprojects {
   }
 }
 
+gradle.projectsEvaluated {
+  subprojects {
+    plugins.withType(JavaPlugin) {
+      getAllDependentProjects(project).each {
+        tasks.javadoc.options.linksOffline "http://linkedin.github.io/gobblin/javadoc/${javadocVersion}/${it.name}/",
```

Comment:
yeah, but we only do releases every once in a while, so this could be linking to javadocs that are weeks old

for now we can leave it as is